### PR TITLE
let some fixed values configurable

### PR DESCRIPTION
**Pull request by KeenS**
## Check list

- [ ] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

## Summary
This p-r make some fixed values specific for certain sensors configurable.

### CODE REVIEW

**Comment by KeenS** [comment]
@ishiy1993 @eguchi1904 do you know any other sensor specific constants used in UnicamIspCapture?

**Comment by ishiy1993** [comment]
:+1:

File: actfw_core/unicam_isp_capture.py [hunk 1]
```diff
@@ -117,62 +108,50 @@ def __init__(
         else:
             raise RuntimeError("Both unicam_size and crop_size must be None or tuples.")
 
-        # control values
+        # config precedence: default < sensor specific < user given
```
Comment (ishiy1993) [inline actfw_core/unicam_isp_capture.py:111]: I think it's better to have no default values. It is better to raise an error if any configuration items are missing.

Comment (KeenS) [reply actfw_core/unicam_isp_capture.py:111]: FYI: libcamera also uses default values
https://github.com/raspberrypi/libcamera/blob/main/src/ipa/raspberrypi/controller/rpi/contrast.cpp#L46

**Review by eguchi1904 — APPROVED** [review]
LGTM

File: actfw_core/unicam_isp_capture.py (hunk 1 above)
Comment (ishiy1993) [reply actfw_core/unicam_isp_capture.py:111]: I see.

**Pull request merged**